webapp/channels/src/components/pricing_modal/content.tsx

```diff
@@ -270,7 +285,7 @@ function Content(props: ContentProps) {
     };
 
     const enterpriseCustomBtnDetails = () => {
-        if (!isPostTrial && isAdmin) {
+        if (!isPostTrial && isAdmin && !cloudFreeDeprecated) {
```

Comment:
@johndavidlugtu once we launch the new flow, will existing cloud starters have that have **not** started an Enterprise trial yet keep having the ability to do so during the 90 day grace period?

Reply:
Good question. Answering the question with a question:

 _If_ we support Cloud Free workspaces to activate an Enterprise trial (like they can right now pre April 27), what additional complexities would we have to deal with during the 90-day period?

@marianunez

Reply:
I will defer to @nickmisasi on what the implications would be given that he did mention there would be additional complexities.

Reply:
Sounds good. Still would want to hear Nick's input, but we should lean towards keeping things simple